private.yaml

```diff
@@ -17,23 +17,39 @@ metadata:
   name: velum-private
   labels:
     name: velum-private
+  annotations:
+    pod.beta.kubernetes.io/init-containers: '[
+      {
+        "name": "mariadb-secrets",
+        "image": "sles12/velum:development",
+        "command": ["sh", "-c", "if [ ! -f /infra-secrets/mariadb-password ]; then head -n 1000 /dev/random | sha256sum | cut -d\" \" -f1 > /infra-secrets/mariadb-password; fi; exit 0;"],
+        "volumeMounts": [
+          {
+            "mountPath": "/infra-secrets",
+            "name": "infra-secrets"
+          }
+        ]
+      }
+    ]'
```

Comment:
i think you can use the spec.initContainers unless we're worried that we'll have to rollback to 1.6 for the api server
https://kubernetes.io/docs/concepts/workloads/pods/init-containers/#support-and-compatibility
like i did in the previous PR https://github.com/kubic-project/caasp-container-manifests/pull/42/files

also, i was busy with bugsquad stuff so i didn't have time to finish the previous PR, if you want i can close that PR and you can implement it, since you're more familiar with the setup?
not trying to dump more work on you tho 😆